modDbImpl/src/org/aion/db/impl/leveldb/LevelDB.java

```diff
@@ -169,6 +169,26 @@ public boolean open() {
         return isOpen();
     }
 
+    public void repair() {
+        LOG.debug("repairing database {}", this.toString());
```

Comment:
should it be the INFO level?

Reply:
It can be. The method will be called infrequently, so there is no risk of spamming the user with unwanted messages. Your call.

Reply:
Then I think we should give it INFO or WARN level, and Change the DB default to WARN level (If we revise this message to WARN)?

Reply:
Ok, will do.